qutebrowser/browser/downloads.py

```diff
@@ -640,6 +640,27 @@ def open_file(self, cmdline=None):
         # (see issue #2296)
         QTimer.singleShot(0, lambda: utils.open_file(filename, cmdline))
 
+    @pyqtSlot()
+    def open_file_dir(self, cmdline=None):
+        """Open the downloaded file's directory.
+
+        Args:
+            cmdline: The command to use as string. A `{}` is expanded to the
+                     filename. None means to use the system's default
+                     application or `downloads.open_dispatcher` if set. If no
+                     `{}` is found, the filename is appended to the cmdline.
+        """
+        assert self.successful
+        filename = (os.path.sep).join(self._get_open_filename().split(os.path.sep)[0:-1])
+        if filename is None:  # pragma: no cover
```

Comment:
This won't work, as calling `.split` on the `None` being returned would already raise an exception. You'll need to do this check before calling anything on the filename.